IFC4x3/Properties/s/SO4 2-_0U5FAS_db4URK3Zf0YMJDR/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SO4 2-_0U5FAS_db4URK3Zf0YMJDR" Name="SO4 2-" UniqueId="1e14f29c-fa79-4479-b503-8e902259335b" PrimaryDataType="IfcReal" />
```

Comment:
SuplhateContent

Reply:
Keeping the names proposed by domain experts.